habitat-hitl/habitat_hitl/config/hitl_defaults.yaml

```diff
@@ -12,6 +12,12 @@ habitat_hitl:
     # Vertical resolution of the window.
     height: 720
 
+  # Font size for text displayed in the GUI app.
+  display_font_size: 16
+
+  # Font to display in the GUI app.
```

Comment:
Any additional instructions for users, if they decide to provide their own font? E.g. are there certain requirements for the ttf file? Will they need to adjust the glyph cache size if they use a much larger font size? I'm just asking for a bit more documentation here.